app/src/repositories/user/userRepository.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const pool = require("../../config/db");
+
+class UserRepository {
+  async findByUserId(user_id) {
```

Comment:
js는 일반적으로 camelCase로 변수를 선언하니 아래처럼 선언하는게 좋을거같아요.

```suggestion
  async findByUserId(userId) {
```

Reply:
대부분의 레이어(controller, service, repository) 대체로 user_id로 사용하는거같은데 전체적으로 변경되면 좋을거같아ㅛ

Reply:
피드백 감사합니다!
원래는 DB 컬럼명과 통일하려고 snake_case로 작성했었는데 말씀해주신 대로 JS 규칙에 맞춰 camelCase로 변경하는 게 더 좋을 것 같네요.
전체적으로 수정해서 반영하겠습니다!